tests/test_datasources.py

```diff
@@ -280,6 +244,43 @@ class SampleData:
                 'warehouse': 'xxxxxxxxlarge',
             },
         ),
+        'explicit-memory-sqlite': DatasourceJSONs(
```

Comment:
These all get tested that pass bootstrapping w/o exception in `test_datasources.py::TestBootstrapDatasource::test_success()`